src/hooks/useSubgraphHealth.ts

```diff
@@ -54,8 +54,6 @@ const useSubgraphHealth = () => {
           `,
         )
 
-        const currentBlock = await simpleRpcProvider.getBlockNumber()
-
```

Comment:
we don't need to check current block, after @chef-jojo 's branch merged to this branch